docs/RouteRequest.md

```diff
@@ -933,34 +1050,59 @@ include stairs as a last result.
 // router-config.json
 {
   "routingDefaults" : {
-    "walkSpeed" : 1.3,
-    "bikeSpeed" : 5,
-    "carSpeed" : 40,
     "numItineraries" : 12,
     "transferPenalty" : 0,
-    "walkReluctance" : 4.0,
-    "bikeReluctance" : 5.0,
-    "bikeWalkingReluctance" : 10.0,
-    "bikeStairsReluctance" : 150.0,
-    "carReluctance" : 10.0,
-    "stairsReluctance" : 1.65,
     "turnReluctance" : 1.0,
     "elevatorBoardTime" : 90,
     "elevatorBoardCost" : 90,
     "elevatorHopTime" : 20,
     "elevatorHopCost" : 20,
-    "escalatorReluctance" : 1.5,
-    "vehicleRental" : {
-      "pickupCost" : 120,
-      "dropOffTime" : 30,
-      "dropOffCost" : 30
+    "bicycle" : {
+      "speed" : 5,
+      "reluctance" : 5.0,
+      "boardCost" : 600,
+      "walk" : {
+        "reluctance" : 10.0,
+        "stairsReluctance" : 150.0
+      },
+      "rental" : {
+        "pickupCost" : 120,
+        "dropOffTime" : "30s",
+        "dropOffCost" : 30
+      },
+      "parking" : {
+        "parkTime" : "1m",
+        "parkCost" : 120
+      },
+      "triangle" : {
+        "safety" : 0.4,
+        "flatness" : 0.3,
+        "time" : 0.3
+      }
+    },
+    "car" : {
+      "speed" : 40,
+      "reluctance" : 10,
+      "decelerationSpeed" : 2.9,
+      "accelerationSpeed" : 2.9,
+      "rental" : {
+        "pickupCost" : 120,
+        "dropOffTime" : "30s",
+        "dropOffCost" : 30
+      },
+      "parking" : {
+        "parkTime" : "5m",
+        "parkCost" : 600
```

Comment:
Can we simplify these two properties even further into `time` and `cost`?

Reply:
At some stage we have to improve the documentation output because we will have several properties called `time`...